Sources/Sentry/Profiling/SentryLaunchProfiling.m

```diff
@@ -135,28 +135,37 @@
 
         SENTRY_LOG_INFO(@"Starting app launch profile at %llu", appLaunchSystemTime);
 
-        // don't worry about synchronizing the write here, as there should be no other tracing
-        // activity going on this early in the process. this codepath is also behind a dispatch_once
-        isTracingAppLaunch = [SentryProfiler startWithTracer:appLaunchTraceId];
+        SentryTransactionContext *context =
```

Comment:
In order to use the required initializer I also have to provide a nameSource. I used `kSentryTransactionNameSourceCustom` since that's what's provided as a default in the cascaded inits in `SentryTransactionContext.m`. I'm not totally clear on what these mean, and all but `kSentryTransactionNameSourceCustom` and `kSentryTransactionNameSourceComponent` appear to be unused in the SDK, so I wasn't sure if I should even add another one or not. Couldn't find anything talking about it on develop.sentry.dev or docs.sentry.io.

I did this in https://github.com/getsentry/sentry-cocoa/pull/3636/files